nservicebus/azure/transactions.md

```diff
@@ -9,56 +9,50 @@ tags:
-
-Such locks become a trust issue in a cloud or self-service environment, as external parties can use these locks to perform a denial of service attack. The Azure platform must assume that you are a malicious user and is thus very hesitant to let you control all the locks by means of a transaction.
+Transaction processing is designed to maintain systems integrity (typically a database or some modern filesystems and services), i.e. to always keep them in a consistent state. That is achieved by ensuring that interdependent operations are either all completed successfully or all canceled. This article focuses on Azure databases and storage services.
 
-This is the primary reason why many Azure hosted services do not support transactions at all or are very aggressive when it comes to lock duration.
+In order to guarantee integrity the database engine must lock a certain number of records inside the transaction when updating values. Which records and how many of them are locked depends, among others, on the selected isolation level. 
 
-For example:
+It is really important to understand, especially in the context of cloud services, that other transactions cannot work with locked records at the same time. In a cloud or self-service environment such locks become a trust issue, because external parties can use them to perform a denial of service attack (sometimes not even intentionally). 
 
- * Azure storage services have no support for transactions. This is not explicitly documented but you can find enough [references on StackOverflow](http://stackoverflow.com/questions/18045517/do-azure-storage-related-apis-participate-in-system-transactions)
- * The Azure database supports local transactions, but only grants locks on resources, when required by a system task for 20 seconds, and 24 hours otherwise. See [Azure SQL Database resource limits](https://azure.microsoft.com/en-us/documentation/articles/sql-database-resource-limits/) for more details.
+This is the primary reason why many Azure hosted services do not support transactions at all or are very aggressive when it comes to the lock duration, for example:
 
-When both the database management system and client are under the same ownership, imagine you just deployed SQL Server to your own virtual machine, so locks are no longer an issue and you can control the lock duration. But even in this case, you need to be careful when it comes to distributed transactions.
+ * Azure storage services have no support for transactions. This is not officially documented, but the information can be found in many places, e.g. [on StackOverflow](http://stackoverflow.com/questions/18045517/do-azure-storage-related-apis-participate-in-system-transactions)
```

Comment:
Would that be ok "Azure Storage Services officially do not participate in transactions. If the transactional behavior is required, it needs to be implemented in the specific system."?